scalafix-rules/src/main/scala/scalafix/internal/rule/RemoveUnused.scala

```diff
@@ -153,6 +153,15 @@ class RemoveUnused(config: RemoveUnusedConfig)
         case Term.Function(List(param @ Term.Param(_, name, _, _)), _)
             if isUnusedParam(name.pos) =>
           Patch.replaceTree(param, "_")
+        case Term.Function(List(param @ Term.Param(_, name, _, _)), _)
+            if isUnusedParam(posExclParens(param)) =>
```

Comment:
Wrapping parens are captured in `param` only when there is a single param. I couldn't find a way to use parens around a param with `implicit` modifier, so I think that's the only place where `posExclParens` is needed...